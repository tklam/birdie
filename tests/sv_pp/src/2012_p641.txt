`define D(x,y) initial $display("start", x , y, "end");

`D( "msg1" , "msg2" )
`D( " msg1", )
`D(, "msg2 ")
`D(,)

