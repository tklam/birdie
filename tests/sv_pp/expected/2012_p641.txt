

initial $display("start", "msg1" , "msg2", "end");
initial $display("start", " msg1" , , "end");
initial $display("start",  , "msg2 ", "end");
initial $display("start",  , , "end");

